rdkit/Chem/Scaffolds/test_MurckoScaffold.py

```diff
@@ -2,352 +2,592 @@
 #
 # Created by Peter Gedeck, June 2008
 #
-from rdkit.Chem.Scaffolds.MurckoScaffold import *
+# pylint: disable=missing-docstring
 
+from collections import namedtuple
+import doctest
 import unittest
-import random
+
 from rdkit import Chem
+from rdkit.Chem.Scaffolds import MurckoScaffold
+from rdkit.Chem.Scaffolds.MurckoScaffold import (GetScaffoldForMol, _pyGetScaffoldForMol,
+                                                 MurckoScaffoldSmilesFromSmiles,
+                                                 MurckoScaffoldSmiles, MakeScaffoldGeneric)
+
+
+TestMolecule = namedtuple('TestMolecule', 'smiles,scaffold')
+
+
+def load_tests(loader, tests, ignore):  # pylint: disable=unused-argument
+  """ Add the Doctests from the module """
+  tests.addTests(doctest.DocTestSuite(MurckoScaffold, optionflags=doctest.ELLIPSIS))
+  return tests
```

Comment:
This is an alternative to using the doctest boilerplate in the module. By adding this function, the doctests are run during as part of the unittest. They are also considered this way as part of a Python coverage run.